hw/ip/otp_ctrl/data/otp_ctrl_scoreboard.sv.tpl

```diff
@@ -940,6 +946,15 @@ class otp_ctrl_scoreboard #(type CFG_T = otp_ctrl_env_cfg)
           end
         end
       end
+      "direct_access_regwen": begin
+        if (addr_phase_write) begin
+          // This permanently locks the DAI.
```

Comment:
"permanently" isn't quite accurate, is it? It locks the DAI until the next power-on reset, right?

Reply:
yeah maybe I shouldn't use permanently here, since that may be confusing in the context of OTP. let me revise this comment.